README.md

```diff
@@ -1,2 +1,17 @@
 # HW01
-Edit the readme
+
+### L. E. Hunter - Biography
+
+Hello, everyone! My name is Laura. I'm an incoming **Integrative Biology** student interested in human evolution. I graduated from Columbia University in 2017 and have been working at a [neuroscience lab](https://www.gottlieblab.com/) at the [Zuckerman Institute for Mind, Brain, and Behavior](https://zuckermaninstitute.columbia.edu/) ever since. In my spare time, I work as an emergency medical technician and volunteer as a fossil and primatology explainer at the American Museum of Natural History (although the museum stuff has been on hold since March due to COVID). I also like to paint. Check out my painting of the skeleton of the animal I studied while working at the Zuckerman Institute:
+
+![Image of Macaque Skeleton](https://user-images.githubusercontent.com/12615998/86480299-bd8b2180-bd1b-11ea-921e-3f4c33582e7f.png)
+
+I'm excited to: 
+
+* take this course this summer and
+* get to know some of you!
+
+The instructions found [here](https://cfss.uchicago.edu/homework/edit-readme/#fn:Perhaps-add-a-pi) says that in order to complete this assignment:
+ >The student describes how they got the changes into README.md and offers a few reflections on their GitHub workflow and their experience with Markdown.
+ 
+It doesn't indicate where to do this, so I guess I will here? These changes were fairly easy to make using the guide linked in the assignment. That made this (my first experience with markdown) pretty easy. I've never worked with GitHub either, but it seems intuitive enough. I guess we will see if I can push this up as easy as it was to pull it down.
```

Comment:
All in all, it looks like you did everything the assignment asked! It's tricky to comment on code clarity and normal rubric stuff since this assignment was pretty straightforward, but good job regardless.